sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -1527,22 +1646,45 @@ export class CertificateClient {
     return this.getCertificateFromCertificateBundle(result._response.parsedBody);
   }
 
-  private getCertificateFromCertificateBundle(certificateBundle: CertificateBundle): Certificate {
+  private getCertificateFromCertificateBundle(
+    certificateBundle: CertificateBundle
+  ): KeyVaultCertificate {
     const parsedId = parseKeyvaultEntityIdentifier("certificates", certificateBundle.id);
 
+    const attributes: any = certificateBundle.attributes || {};
     const policy = toPublicPolicy(certificateBundle.policy || {});
 
-    const properties = {
+    let abstractProperties: any = {
+      keyId: certificateBundle.kid,
+      secretId: certificateBundle.sid,
+      name: parsedId.name,
+      createdOn: attributes.created,
+      updatedOn: attributes.updated,
+      expiresOn: attributes.expires,
       ...parsedId,
       ...certificateBundle,
       ...certificateBundle.attributes
```

Comment:
This `...certificateBundle.attributes` results in `recoveryLevel` and `enabled` being extracted out which is not present in the definition for `CertificateProperties`. This behavior existed before this PR as well, so create an issue for a follow up.

Reply:
This has been handled in @jonathandturner's commit https://github.com/Azure/azure-sdk-for-js/pull/5991/commits/1ef22d22baf363462e791bd9064546c1208a0bdf